.travis.yml

```diff
@@ -4,6 +4,6 @@ git:
 sudo: false
 language: node_js
 node_js:
-  - '4'
-  - '6'
-  - '7'
+  - '8'
+  - '9'
```

Comment:
9 is EOL; you should be testing on 4/6/8/10 (and you should only drop 4 if babel has)